models/shipping-method/src/shipping-method/transformers.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  LocalizedString,
+  Reference,
+  type TReferenceGraphql,
+} from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type {
+  TShippingMethod,
+  TShippingMethodRest,
+  TShippingMethodGraphql,
+} from './types';
+
+const transformers = {
+  default: Transformer<TShippingMethod, TShippingMethod>('default', {
+    buildFields: [
+      'localizedName',
+      'localizedDescription',
+      'taxCategory',
+      'zoneRates',
+      'createdBy',
+      'lastModifiedBy',
+    ],
+  }),
+  rest: Transformer<TShippingMethod, TShippingMethodRest>('rest', {
+    buildFields: [
+      'localizedName',
+      'localizedDescription',
+      'zoneRates',
+      'createdBy',
+      'lastModifiedBy',
+      'taxCategory',
+    ],
+    replaceFields: ({ fields }) => ({
+      ...fields,
+      taxCategory: Reference.random().typeId(fields.taxCategory.id).buildRest(),
```

Comment:
```suggestion
      taxCategory: Reference.random().id(fields.taxCategory.id).typeId('tax-category`).buildRest(),
```
Is the `buildRest` needed here?